backend/pages/utils/chart_classification.py

```
@@ -1,3 +1,5 @@
+from .advanced_processing import process_balance_sheet, process_income_statement, process_detailed_brand_demand, process_compensation, process_competitor_city, process_production_costs, process_cashflow
```

Ruff (0.11.9)

`.advanced_processing.process_compensation` imported but unused

Remove unused import: `.advanced_processing.process_compensation`

(F401)